english/check-phone/open-message/open-message.md

```diff
@@ -3,3 +3,10 @@ white. Two black dots appear, slowly morphing into eyes as the white of
 the screen takes over your whole field of vision. 
 
 The marshmallows are here.
+What do you do?
+
+[Become a marshmallow.](https://github.com/mjjellybean02/create-your-own-adventure/blob/master/english/becomemarshmallow/becomemarshmallow.md)
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.